HPXMLtoOpenStudio/measure.rb

```diff
@@ -335,6 +335,16 @@ def self.set_defaults_and_globals(runner, output_dir, epw_file, weather, schedul
     # Now that we've written in.xml, ensure that no capacities/airflows
     # are zero in order to prevent potential E+ errors.
     HVAC.ensure_nonzero_sizing_values(@hpxml)
+
+    # Handle zero occupants when operational calculation
+    occ_calc_type = @hpxml.header.occupancy_calculation_type
+    noccs = @hpxml.building_occupancy.number_of_residents
+    if occ_calc_type == HPXML::OccupancyCalculationTypeOperational && noccs == 0
+      @hpxml.header.vacancy_periods.add(begin_month: 1,
+                                        begin_day: 1,
+                                        end_month: 12,
+                                        end_day: 31)
```

Comment:
I assume this works even if the simulation period is not 1/1-12/31? Maybe it would be better to set this identical to the simulation period?

Reply:
I will set identical to the simulation period. Also, I will add a sample file.